### PR TITLE
styles: add informative tooltips to the Sponsor Dashboard stats section

### PR DESCRIPTION
**Pull request by mb-008**
## What does this PR do?
This PR tends to solve the issue of  adding tooltips to the sponsor dashboard stats. I have added tooltip to each stat providing subtle details . 

## Where should the reviewer start?
The reviewer can begin & finish in the banner file itself as i have used the components from chakra-ui only in it . 

## How should this be manually tested?
By hovering over the sponsor profile you can manually test it. 

## Any background context you want to provide?
Nope

## What are the relevant issues?
#415 

## Screenshots (if appropriate)
tt

### CODE REVIEW

**Review by a20hek — CHANGES_REQUESTED** [review]
should ideally add a i/infotip logo next to the headers "rewarded", "listings", "submissions", and hovering on them should show the following: 

Rewarded: Total compensation (in USD) of listings where the winners have been announced 

Listings: Total number of listings added to Earn 

Submissions: Total number of submissions/applications received on all listings

**Comment by a20hek** [comment]
@mb-008 are you going to be doing the changes

**Comment by mb-008** [comment]
> @mb-008 are you going to be doing the changes

I am facing some issues with the dashboard/listing route & unable to debug it

**Comment by a20hek** [comment]
> > @mb-008 are you going to be doing the changes
> 
> I am facing some issues with the dashboard/listing route & unable to debug it

and what are those issues?

**Comment by mb-008** [comment]
> > > @mb-008 are you going to be doing the changes
> > 
> > 
> > I am facing some issues with the dashboard/listing route & unable to debug it
> 
> and what are those issues?
Server Error
TypeError: Endpoint URL must start with `http:` or `https:`.

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source
src\utils\contract\contract.ts (14:26) @ eval

  12 | const RPC_URL = process.env.NEXT_PUBLIC_PAYMENT_RPC_URL || '';
  13 |
> 14 | export const connection = new anchor.web3.Connection(RPC_URL, 'confirmed');
     |                        ^
  15 |
  16 | const getProvider = (wallet: anchor.Wallet) => {
  17 | const opts = {

This is the error i am facing

**Comment by mb-008** [comment]
Screenshot 2024-03-20 090204

did all the requested changes

File: src/utils/contract/contract.ts [hunk 1]
```diff
@@ -9,7 +9,9 @@ import { LAMPORTS_PER_SOL, PublicKey } from '@solana/web3.js';
 import { type EarnReloaded, IDL } from './program';
 
 const PROGRAM_ID = process.env.NEXT_PUBLIC_PAYMENT_PROGRAM_ID || '';
-const RPC_URL = process.env.NEXT_PUBLIC_PAYMENT_RPC_URL || '';
+const RPC_URL =
```
Comment (a20hek) [inline src/utils/contract/contract.ts:12]: why would you change this file?
please revert changes in this file and pull latest code from main

**Review by a20hek — CHANGES_REQUESTED** [review]

File: src/utils/contract/contract.ts (hunk 1 above)
Comment (mb-008) [reply src/utils/contract/contract.ts:12]: Due to this change only my dasboard/listings seems to be working . It was  how i debugged the error. But i will change the required code.

**Pull request merged**